packages/php-wasm/universal/src/lib/php.ts

```diff
@@ -591,9 +606,17 @@ export class PHP implements Disposable {
 		let heapBodyPointer: number | undefined;
 		const streamedResponsePromise = this.#executeWithErrorHandling(
 			async () => {
-				if (!this.#webSapiInitialized) {
-					await this.#initWebRuntime();
-					this.#webSapiInitialized = true;
+				if (!this.#phpWasmInitCalled) {
```

Comment:
I inlined this call to reduce the mental load – there were too many names for me to follow